sentry-core/src/main/java/io/sentry/core/SentryEvent.java

```diff
@@ -196,6 +197,13 @@ public void setSdk(SdkVersion sdk) {
     return fingerprint;
   }
 
+  public boolean hasFingerprint(final @NotNull String fingerprint) {
```

Comment:
h: Not sure this makes sense since the Fingerprint is the whole list, not a single item in it.
What's the motivation for this API?

Reply:
If one wants to filter it, it would be possible, the getter is not public, what would you suggest? Returning the whole list?

Reply:
Yeah that'd be the way to go. Just `List<> getFingerprint` looks good